remove-null-values/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    function removeNullValues(array) {
+    var result = []
+
+      for (i = 0; i < array.length; i++) {
+        var value = array[i];
+        if (value) {
+          result.push(array[i])
+        }
```

Comment:
This won't just remove null values, it will remove all falsy values. Please fix.